src/Commands/Cap.cpp

```
@@ -0,0 +1,92 @@
+	std::string::size_type spacePos = params.find(" ");
+	std::string subcommand;
+	std::string args;
+	if (spacePos == std::string::npos) {
+		subcommand = params;
+	} else {
+		subcommand = params.substr(0, spacePos);
+		args = params.substr(spacePos + 1);
+	}
+	//Convert subcommand to uppercase for case-insensitive matching
+	std::string subcommandUpper = subcommand;
+	for (std::string::iterator it = subcommandUpper.begin(); it != subcommand.end(); ++it) {
```

Cppcheck (2.10-2)

[error] Iterators of different containers 'subcommandUpper' and 'subcommand' are used together.

(mismatchingContainers)